tensorpipe/test/channel/channel_test.cc

```diff
@@ -208,8 +203,7 @@ TEST_P(ChannelTest, SendMultipleTensors) {
         // Validate contents of vector.
         for (auto& data : dataVec) {
           for (int i = 0; i < data.size(); i++) {
-            uint8_t value = i;
-            EXPECT_EQ(data[i], value);
+            EXPECT_EQ(data[i], (uint8_t)i);
```

Comment:
Let's follow Google's Style Guide here: https://google.github.io/styleguide/cppguide.html#Casting.

Reply:
We've never really defined what style guide we follow, but we're certainly not sticking consistently to Google's one ;)

In this case here, though, I'm fine, because it echoes what I said:

> Do not use cast formats like (int)x unless the cast is to void.